### ISSUE TITLE
sun dial | spins uncontrollably

### ISSUE THREAD

**Issue by leoauri** [agnibho/solarcompass#1]
Hi
On my Fairphone FpOS 1.8.7 android 4.2.2
Trying to rotate the sun knob results in uncontrollable spinning of it
Cannot set the position of the sun
Great idea!!

**Comment by agnibho** [comment]
Thank you, I'll see if I can reproduce the problem.

**Comment by robinpaulson** [comment]
Yep, I get this too. Replicant 4.2 on Samsung Galaxy S3. More precisely, as I try to set the position of the sun, it moves very erratically, usually not going where I click. Dragging it round the circle results in very odd almost random movement.

Cheers

**Comment by LavirtheWhiolet** [comment]
Pocketbook SURFpad, Android 4.4.2, the same problem. Also the compass image floats randomly.
